src/language/seq.md

```diff
@@ -0,0 +1,33 @@
+# Sequences and arrays
+
+## Sequences
+
+The staple `Vec` type is forbidden in hacspec. Instead, you have to use the
+type `Seq`, which is implemented as a wrapper around `Seq`.
+
+The most notable differences between `Seq` and `Vec` is that `Seq` is not
+resizable, and does not support `push` and `pop` operations. Instead, the
+final length of the seq has to be provided at creation time. See the
+hacspec standard library documentation for more details.
+
+`Seq` is a generic type that always has to be indexed by the content of the
+cells: `Seq<u8>`, etc.
```

Comment:
You say earlier that generics aren't allowed. But it is here again for Seq. I think this needs clarification somewhere.